tidb-cloud/v7.1.0-performance-benchmarking-with-sysbench.md

```diff
@@ -0,0 +1,166 @@
+summary: Introduce the Sysbench performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud Sysbench Performance Test Report
+
+This document provides the Sysbench performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB v7.1.0 clusters.
+
+## Test overview
+
+This test aims at showing the Sysbench performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+### TiDB cluster
+
+The test is conducted on a TiDB cluster with the following settings:
+
+- Cluster type: [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier)
+- Cluster version: v7.1.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+
+### Parameter configuration
+
+   Enabling the below parameter can have a positive impact on performance in the 'oltp_insert','oltp_read_write','oltp_update_index','oltp_update_non_index' scenario.
+
+   Disabling this parameter can potentially improve performance in the 'oltp_point_select' scenario. The 'raft-engine.prefill-for-recycle' parameter is disabled by default.
+
+#### TiKV parameter configuration
+
+{{< copyable "" >}}
+
+```yaml
+raft-engine.prefill-for-recycle = true
+```
+
+#### How to edit tikv config
+we recommend contacting our support team. Our support team is available to assist you with parameter modifications according to your specific requirements.
+
+### Benchmark executor
+
+The benchmark executor sends SQL queries to the TiDB cluster. In this test, its hardware configuration is as follows:
+
+- Machine type: Amazon EC2 (us-west-2)
+- Instance type: c6a.2xlarge
+
+## Test steps
+
+This section introduces how to perform the Sysbench performance test step by step.
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Dedicated cluster that meets the [test environment](#tidb-cluster) requirements.
+
+   For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).
+
+2. On the benchmark executor, connect to the newly created cluster and create a database named `sbtest`.
+
+   To connect to the cluster, see [Connect via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md).
+
+   To create the `sbtest` database, execute the following SQL statement:
+
+   ```sql
+   CREATE DATABASE sbtest;
+   ```
+
+3. Load Sysbench data to the `sbtest` database.
+
+   1. The test in this document is implemented based on [sysbench](https://github.com/akopytov/sysbench). You can download the test program using the following command:
+
+      ```shell
+      curl -s https://packagecloud.io/install/repositories/akopytov/sysbench/script.deb.sh | sudo bash
```

Comment:
This script cannot install the latest version of sysbench. I think we should recommend the user to use "Build and Install" to install sysbench and also provide the sysbench version we used